src/main/resources/pipeline/github/github.yaml.mustache

```diff
@@ -0,0 +1,48 @@
+
+# TODO: after append all the tasks require for your project,
+# you need to move this pipeline to .github/workflows directory if doesn't exist you will must be create.
```

Comment:
if this file should be in a specific path, I suggest you should put it in that location by default

Reply:
you're right. solved